docs/projects.md

```diff
@@ -39,3 +40,42 @@ dronedeployApi.Projects.getCurrentlyViewed()
 ```
 
 [**Full Example**](/projects/example-projects.getcurrentlyviewed.md)
+
+### Projects.team
+
+**Overview**
+
+Returns members who have access to this project.
+
+**Example Call**
+
+```javascript
+// Get the current one time
```

Comment:
I doesn't sound right. I guess that you wanted to say `Get member for current project one time`? Maybe try this one instead